service/popm/popm.go

```diff
@@ -110,10 +123,11 @@ func NewMiner(cfg *Config) (*Miner, error) {
 
 	m := &Miner{
 		cfg:            cfg,
-		keystoneCh:     make(chan *hemi.L2Keystone, 3),
 		bfgCmdCh:       make(chan bfgCmd, 10),
 		holdoffTimeout: 5 * time.Second,
 		requestTimeout: 5 * time.Second,
+		mineNowCh:      make(chan struct{}, 1),
+		l2Keystones:    make(map[string]L2KeystoneProcessingContainer),
```

Comment:
These really should have an initial size

Reply:
yeah I can do that, I can make it the size of the max keystones (10)

Reply:
done 👍